.github/workflows/cve-scanning-node.yml

```diff
@@ -0,0 +1,57 @@
+name: CVE Scanning for Node.js
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 8,18 * * 1-5'
+  push:
+    paths:
+      - '**/package.json'
+      - '**/package-lock.json'
+      - '.github/workflows/node-cve-ignore-list.xml'
+      - '.github/workflows/cve-scanning-node.yml'
+
+jobs:
+  node-modules-scan:
+    name: ${{ matrix.module-folder }}-node-scan
+    runs-on: ubuntu-latest
+    continue-on-error: false
+    strategy:
+      matrix:
+        module-folder: [
+          'cli',
+          'docs',
+          'visualizer' ]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Set up Node
+        uses: actions/setup-node@v3
```

Comment:
checkout/setup versions are v3 here, v4 in other file

Reply:
Bumped them all to v4